contributors/devel/release/patch_release.md

```diff
@@ -0,0 +1,87 @@
+# Building Kubernetes patch releases
+
+This document describes the process for creating Kubernetes patch releases and communicating publishing
+status regarding known issues impacting the current release.
+
+## Process
+
+After a Kubernetes minor release, patch releases are managed by the Kubenretes patch release czar.
+The patch release czar works with SIGs to coordinate merging cherrypicks into the release branch,
+then cutting and publishing a new patch release.
+
+## Communicating fixes to the community
+
+The canonical source of issues in a patch release will be communicated through GitHub issues.
+
+To make it easy for the community to understand the status of each patch release, GitHub issues
+must follow a consistent format and contain basic information about the issue.
+
+GitHub issues targeted at patch releases must contain the following items:
+
+### Release label
+
+The release label contains information about which releases are impacted by the issue, and
+which release the issue is resolved in.
+
+- `release/introduced-in/1.7.0`
+- `release/resolved-in/1.7.2`
```

Comment:
I get the intent and I can't think of a better way, but it does feel like opening the door to drowning in labels.  This implies as followup:
- `release/introduced-in/x.y.z` label created as part of each release
- `release/resolved-in/x.y.z` created as part of each release
- a `/release` bot command to allow non-members to add these labels?
- verification that each issue to which each cherry-pick PR is linked has these labels... I'm guessing this is a small enough set that release managers can verify this by hand?

the semver bikeshed nerd in me is wondering if these should be `release/foo-in/v1.2.3`